lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -799,17 +815,24 @@ def generate_docker_py_service_description(self, name, docker_networks):
                 )
             )
 
+        dns_config = types.DNSConfig(
+            nameservers=self.dns,
+            search=self.dns_search,
+            options=self.dns_options
+        )
+
         cspec = types.ContainerSpec(
             image=self.image,
-            user=self.user,
```

Comment:
How about ordering the arguments alphabetically? :)

Reply:
Most or all of the arguments to other initialisers of docker types are passed in the same order as they are defined in the implementation. In my opinion that's a pretty good practice to follow.

Reply:
That's also fine for me. Then let's hope they don't reorder them ;-)